packages/mirage-http/mirage-http.2.3.0/opam

```diff
@@ -0,0 +1,20 @@
+opam-version: "1.2"
+maintainer:   "anil@recoil.org"
+authors:      ["Anil Madhavapeddy" "Thomas Gazagnaire"]
+homepage:     "https://github.com/mirage/mirage-http"
+bug-reports:  "https://github.com/mirage/mirage-http/issues/"
+dev-repo:     "https://github.com/mirage/mirage-http.git"
+
+build:    [make]
+install: [make "install"]
+remove:  ["ocamlfind" "remove" "mirage-http"]
+depends: [
+  "ocamlfind" {build}
+  "mirage-types-lwt" {>= "2.0.0"}
+  "mirage-conduit"   {>= "2.2.0"}
+  "lwt" {>= "2.4.3"}
+  "cohttp" {>= "0.16.0" & < "0.18.0"}
+  "camlp4"
```

Comment:
i think the camlp4 should actually be type_conv here, to ensure sexplib.syntax is available